Formula/elan-init.rb

```diff
@@ -37,20 +35,22 @@ def install
   end
 
   test do
-    system bin/"elan-init", "-y"
+    ENV["ELAN_HOME"] = testpath/".elan"
+
+    system bin/"elan-init", "-y", "--default-toolchain=leanprover/lean4:nightly"
```

Comment:
Do we need to use the nightly toolchain for testing here?

Reply:
`--default-toolchain=leanprover/lean4:stable` will also do. The latest stable is a bit outdated but indeed it shouldn't matter that much for the purpose of this test.